flocker/restapi/docs/publicapi.py

```diff
@@ -420,7 +407,13 @@ def run(self):
 
         appContainer = namedAny(self.arguments[0])
 
-        examples_path = FilePath(self.options["examples_path"])
+        # For now we ignore this and just hardcode, due to
+        # https://github.com/sphinx-doc/sphinx/issues/1685
+        # examples_path = FilePath(self.src).preauthChild(
+        #     options["examples_path"]))
```

Comment:
Looks like it does the right thing.  To make it easier to use this, I've also changed the value expected by the `examples_path` option.  It's relative to the source file it's in, now, instead of relative to the root of the repository.